src/helper/signupEmail.js

```diff
@@ -0,0 +1,22 @@
+import sgMail from '@sendgrid/mail';
+import { config } from 'dotenv';
+
+config();
+
+const signupVerificationMail = (userEmail, emailToken) => {
+  const confirmationUrl = `process.env.BASE_URL/confirmemail?email=${userEmail}&token=${emailToken}`;
```

Comment:
`BASE_URL` should be resolved from the `.env` configuration.

Reply:
5 days!!!

Reply:
@kayroy247

Reply:
I have modified the email sender module so that it can be used anywhere in the app so the BASE_URL environment variable is no more there. Thanks